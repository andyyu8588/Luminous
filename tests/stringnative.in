function mySubstring(str) {
    return substring(str, 3, 5);
}

function wrongSubstring(str) {
    return substring(str, 5, 5);
}

print(mySubstring("ok"));
print(mySubstring("okbn"));
print(mySubstring("hello world!"));
print(wrongSubstring("blablablabla"));
print(size(substring("nice to meet you!", 0, 8)));

print("nani" < "nano");
print("andys" < "andy");
print("okkk" > "okka");