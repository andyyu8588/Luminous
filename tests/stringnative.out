
n
lo

8
true
false
true
